### ISSUE TITLE
fuck this

### ISSUE THREAD

**Issue by syrupderg** [bound-mod/bound#1]
i was genuinely waiting an update from you for vendetta since it had some bugs but you spend your time and effort just to make an april fools joke.

**Comment by GhostOfJericho72** [comment]
Yeah. The way they've acted and treated a server with 36,000 people is fucked. 
An April fools joke is meant to be funny. 
Disappointing behaviour

**Comment by Compwnter** [comment]
im going to dip my nutsack in marinara sauce

**Comment by NickHam13** [comment]
> i was genuinely waiting an update from you for vendetta

you don't know how to read lil bro

**Comment by jackmthws** [comment]
It was made pretty clear Vendetta was EOL'd. God forbid the devs have a little fun with their defunct project.

**Comment by GhostOfJericho72** [comment]
Yes they made the eol message clear; however with the constant possibility of projects being resurrected, getting up to 36,000 peoples hopes up (that don't realise it's April 1st), and then letting them down is pathetic. 
I hope they grow up.

**Comment by jackmthws** [comment]
womp womp

**Comment by danilwhale** [comment]
pluh

**Comment by LemonyOwO** [comment]
Womp womp

**Comment by yellowsink** [comment]
> Yes they made the eol message clear; however with the constant possibility of projects being resurrected, getting up to 36,000 peoples hopes up (that don't realise it's April 1st), and then letting them down is pathetic. I hope they grow up.

well these 36000 people who are apparently hoping for a resurrection can learn to read then

**Comment by wingio** [comment]
This was the most obvious april fools joke someone could make, the name even clearly referencing a different client mod that many vendetta users know about

**Comment by Vendicated** [comment]
paypal maisy £36000 for insta vendetta resurrection

**Comment by mugman174** [comment]
Don't forget: Vendetta will return on 2 April

**Comment by amyulating** [comment]
skull issue

**Comment by Overimagine1** [comment]
> Yeah. The way they've acted and treated a server with 36,000 people is fucked. An April fools joke is meant to be funny. Disappointing behaviour

ik im late but you're boring as hell

the client mod that got discontinued for personal reasons.... getting continued again on april fools is a prank???? wow!!!
